### PR TITLE
Add compat data for WorkerNavigator (Worker API)

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/WorkerNavigator

### CODE REVIEW

File: api/WorkerNavigator.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
+{
+  "api": {
+    "WorkerNavigator": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/WorkerNavigator",
+        "support": {
+          "webview_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/WorkerNavigator.json]: `true`

File: api/WorkerNavigator.json [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+          },
+          "chrome": {
+            "version_added": null
```
Comment (jpmedley) [inline api/WorkerNavigator.json]: `"1"`

File: api/WorkerNavigator.json [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/WorkerNavigator.json]: `true`

File: api/WorkerNavigator.json [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/WorkerNavigator.json]: `true` for both versions of Opera.

File: api/WorkerNavigator.json [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": "53"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
```
Comment (jpmedley) [inline api/WorkerNavigator.json]: `true`

File: api/WorkerNavigator.json [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WorkerNavigator/sendBeacon",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/WorkerNavigator.json]: 39 for all 3

File: api/WorkerNavigator.json [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/WorkerNavigator.json]: 26 for both

File: api/WorkerNavigator.json (hunk 1 above)
Comment (maboa) [reply api/WorkerNavigator.json]: 👍

File: api/WorkerNavigator.json (hunk 2 above)
Comment (maboa) [reply api/WorkerNavigator.json]: 👍

File: api/WorkerNavigator.json (hunk 3 above)
Comment (maboa) [reply api/WorkerNavigator.json]: 👍

File: api/WorkerNavigator.json (hunk 4 above)
Comment (maboa) [reply api/WorkerNavigator.json]: 👍

File: api/WorkerNavigator.json (hunk 5 above)
Comment (maboa) [reply api/WorkerNavigator.json]: 👍

File: api/WorkerNavigator.json (hunk 6 above)
Comment (maboa) [reply api/WorkerNavigator.json]: 👍

File: api/WorkerNavigator.json (hunk 7 above)
Comment (maboa) [reply api/WorkerNavigator.json]: 👍

**Comment by maboa** [comment]
Those changes should be made now :)

**Pull request merged**